src/examples/Dns/Generated/Operations/RecordSetsOperations.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Dns.Models.V20180501;
+
+namespace Azure.Dns.Operations.V20180501
+{
+    public static class RecordSetsOperations
+    {
+        public static async ValueTask<Response<RecordSet>> UpdateAsync(HttpPipeline pipeline, string resourceGroupName, string zoneName, string relativeRecordSetName, RecordType recordType, string apiVersion, string subscriptionId, RecordSet parameters, string? ifMatch = default, CancellationToken cancellationToken = default)
+        {
+            //using ClientDiagnostics scope = pipeline.CreateScope("Azure.Dns.Operations.V20180501.Update");
+            //scope.AddAttribute("key", name)
+            //scope.Start()
+            try
+            {
+                var request = pipeline.CreateRequest();
+                request.Method = RequestMethod.Patch;
+                //request.Uri.Reset();
+                if (ifMatch != null) { request.Headers.SetValue("If-Match", ifMatch.ToString() ?? string.Empty); }
+                request.Uri.AppendQuery("api-version", apiVersion.ToString() ?? string.Empty);
```

Comment:
`apiVersion.ToString() ?? string.Empty` expression doesn't make sense

Reply:
`ToString()` returns `string?`. You pointed that out to me, remember?
https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Object.cs#L39

Reply:
File removed.